middle_end/flambda/types/structures/row_like.rec.ml

```diff
@@ -391,19 +442,19 @@ struct
         | At_least index -> At_least (Index.apply_renaming index renaming)
       in
       let known_tags' =
-        Tag.Map.map_sharing (fun { index; maps_to } ->
-            { index = rename_index index;
-              maps_to = Maps_to.apply_renaming maps_to renaming;
-            })
+        Tag.Map.map_sharing (fun { index; maps_to; env_extension; } ->
+          let env_extension = TEE.apply_renaming env_extension renaming in
```

Comment:
Style remark: I don't really see why `env_extension` is let-bound but `maps_to` is not. Could you make them consistent ?